CODEOWNERS

```diff
@@ -40,7 +40,7 @@ game-servers @GoogleCloudPlatform/dee-platform-ops @GoogleCloudPlatform/nodejs-s
 monitoring @GoogleCloudPlatform/dee-platform-ops @GoogleCloudPlatform/nodejs-samples-reviewers
 
 # SoDa teams
-cloud-sql @GoogleCloudPlatform/infra-db-dpes @GoogleCloudPlatform/nodejs-samples-reviewers
+cloud-sql @GoogleCloudPlatform/infra-db-sdk @GoogleCloudPlatform/nodejs-samples-reviewers
```

Comment:
@ttosta-google probably need the team to be granted write access first as its showing an error.

Reply:
Thanks @jackwotherspoon for checking it. I confirmed with Eno that we don't have admin access to this repo so the reviewer should fix it after the PR is merged.

Reply:
I've provided write access to your team, so I think we're good to go now?